run_aquarium.sh

```diff
@@ -23,12 +23,14 @@ has_config=false
 config_path=""
 is_new=false
 with_systemdeps=false
+port=1337
 
 while [[ $# -gt 0 ]]; do
   case $1 in
     -d|--debug) is_debug=true ;;
     -c|--config) has_config=true ; config_path=$2 ; shift 1 ;;
     -n|--new) is_new=true ;;
+    -p|--port) port=$2 ; shift 1 ;;
```

Comment:
this needs a check for whether `${port}` is populated afterwards. Something like `./run_aquarium --port` or `./run_aquarium --port ""` would end up with an empty variable.